README.md

```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman 
+
+## Building Manually
+To build manually, you are going to need the following dependentcies
+
+pacman fakeroot git binutils npm electron(maunaully downloaded from github, see later) bzip2 make gcc
```

Comment:
```suggestion
`pacman fakeroot git binutils npm electron bzip2 make gcc`
```

Reply:
The PKG build looks specifically for a folder named electron 22, when the current version is 24